src/components/Button/InputButton.test.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { create } from "react-test-renderer";
+import { mount } from "enzyme";
+
+import { InputButton } from "./InputButton";
+
+jest.useFakeTimers();
+
+it("renders correctly with all props", () => {
+  const component = create(
+    <InputButton
+      id="1234"
+      className="class1 class2"
+      isStartButton
+      preventDoubleClick
+      name="Testname"
+      type="submit"
+      value="Testvalue"
+      disabled
+    />
+  );
+
+  expect(component).toMatchSnapshot();
+});
+
+it("renders correctly without optional props", () => {
+  const component = create(<InputButton type="submit" />);
+
+  expect(component).toMatchSnapshot();
+});
+
+it("prevents onClick from being called twice within a second when preventDoubleClick is true", () => {
+  const wrapper = mount(
+    <InputButton preventDoubleClick type="submit">
+      Testchildren
+    </InputButton>
+  );
+
+  const input = wrapper.find("input");
+  const inputOnClick = input.props().onClick;
+
+  expect(inputOnClick).toBeDefined();
+
+  const mockPreventDefault = jest.fn();
+  const mockClickEvent = {
+    ...jest.fn()(),
+    target: input.getDOMNode(),
+    preventDefault: mockPreventDefault
+  };
+
+  //eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  inputOnClick!.call(input, mockClickEvent);
+
+  expect(mockPreventDefault).toHaveBeenCalledTimes(0);
```

Comment:
```suggestion
  expect(mockPreventDefault).not.toHaveBeenCalled();
```

This is more readable. Only change if you're here anyway.